typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -319,8 +319,33 @@ class SyncLogViewer extends Component<SyncLogViewerProps, State> {
         nextProps: SyncLogViewerProps /*, nextState: State*/
     ): void {
         // called before render()
-        if (this.props.welllogs.length !== nextProps.welllogs.length)
-            this.fillViewCallbacks(nextProps.welllogs.length); // update this.callbacks[] before render()
+        if (this.props.welllogs.length === nextProps.welllogs.length) return;
+        if (this.controllers.length === nextProps.welllogs.length) return;
+        /*
+        // move old controllers to new places in the controllers array
```

Comment:
what is the purpose of this commented out code?

Reply:
It was an attempt of some optimization but currentrly it does not work properly